packages/shopify-app-remix/docs/staticPages/billing.doc.ts

```diff
@@ -0,0 +1,89 @@
+import {LandingTemplateSchema} from '@shopify/generate-docs';
+
+const data: LandingTemplateSchema = {
+  id: 'guide-billing',
+  title: 'Billing for your app',
+  description:
+    'You can bill merchants for your app via Shopify using the Admin API, either as a one-time purchase or as a recurring subscription.' +
+    '\n\nShopify will handle the transactions, based on purchases set up by the app and agreed upon by the merchant.' +
+    '\n\nFor more information on how this process works, see [the billing documentation](/docs/apps/billing).',
+  sections: [
+    {
+      type: 'Generic',
+      anchorLink: 'plans',
+      title: 'Setting up plans',
```

Comment:
It seems like outside of libriaries we don't really use the term "plans". Instead it is [App Purchases.](https://shopify.dev/docs/apps/billing/subscriptions)